pom.xml

```diff
@@ -118,8 +154,17 @@
                 <plugin>
                     <groupId>org.jenkins-ci.tools</groupId>
                     <artifactId>maven-hpi-plugin</artifactId>
+                    <extensions>true</extensions>
                     <configuration>
-                        <pluginFirstClassLoader>true</pluginFirstClassLoader>
+                        <pluginFirstClassLoader>false</pluginFirstClassLoader>
```

Comment:
Hose tickets seem related to the guava version being loaded, isn't it? With this PR, guava keeps being fixed at guava 17, with the logger reporting suche version being loaded, both for pipelines and "manual" provisioning.
What further tests would you expect to perform?